README.md

```
@@ -1,55 +1,246 @@
+
+## 🎯 Target Users
+- Parents and guardians of babies and children aged **0 months to 10 years**.
+- Caregivers who want a centralized baby care tracking solution.
+
+---
+
+## 🖼️ UI/UX Showcase
+
+Here's a glimpse of our user-friendly interface that supports parents with clarity, warmth, and ease of use:
+
+<img src="https://raw.githubusercontent.com/Learnathon-By-Geeky-Solutions/triangle/refs/heads/feature/vaccination/assets/images/LittleSteps%20Scratch.png" alt="UI/UX Design Preview" width="400"/>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)